en/cloud_development_setup/instructions.md

```diff
@@ -0,0 +1,24 @@
+You can [skip right over this section](http://tutorial.djangogirls.org/en/installation/#install-python) 
```

Comment:
This as a starting sentence doesn't make sense, I would not begin a page with "You can skip right over this section" as then there's not point it being there?

Reply:
Ah! I've realised what it should be due to the second sentence - amend to "If you have already worked through the installation steps, you do not need to also complete the RunCode Cloud Environment setup. Please skip straight to Introduction to Python"